hydrophone.go

```diff
@@ -146,6 +146,8 @@ func main() {
 		mail, mailErr = sc.NewSesNotifier(&config.Ses)
 	case "smtp":
 		mail, mailErr = sc.NewSmtpNotifier(&config.Smtp)
+	case "null":
+		mail, mailErr = sc.NewNullNotifier()
 	default:
 		log.Fatalf("the mail system provided in the configuration (%s) is invalid", config.NotifierType)
 	}
```

Comment:
I was wondering whether the name "mail client" is accurate here? Should it be "mail server" instead? Or maybe "protocol"?

Reply:
also, would that make sense to have the log.Printf("Success: email sent") thing found in hydrophone.go (line 287) in each notifier that is actually sending emails (in the "Send" method) (except for the NullNotifier)

Reply:
> 
> 
> I was wondering whether the name "mail client" is accurate here? Should it be "mail server" instead? Or maybe "protocol"?

I don't think so, wince this is the hydrophone act like an email client.

Reply:
> 
> 
> also, would that make sense to have the log.Printf("Success: email sent") thing found in hydrophone.go (line 287) in each notifier that is actually sending emails (in the "Send" method) (except for the NullNotifier)

I don't see the reason. I'm not sure to understand what you want here.
Which message do you think of ?